src/views/HomeView.vue

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="row">
       <div class="col">
-        Hello
```

Comment:
I updated this line because it wasn't super helpful. Tried to give more information.